packages/application-components/src/components/main-pages/internals/main-page.styles.ts

```diff
@@ -15,6 +22,12 @@ export const MainPageContent = styled.div`
   flex: 1;
   flex-basis: 0;
   overflow: auto;
-  // NOTE: do not change to "padding" as this breaks sticky DataTable styles
+  // NOTE: do not change to "margin" as this breaks sticky DataTable styles
+  // Bear in mind the margins are dependant on the context the component
+  // is rendered in. For example, when rendered in a custom view, its panel
+  // already contains default paddings, so we don't need to add margins here.
```

Comment:
Here as well

Reply:
I forgot about that one. Sorry.

Updated here: 64f6bef6a0578c2b403344243f5b566d520f363b